dashboard/app/models/course_offering.rb

```diff
@@ -457,4 +467,29 @@ def high_school_level?
 
     true
   end
+
+  def get_available_resources(locale_code='en-us')
+    lessons = latest_published_version(locale_code).units.first.lessons
+
+    return nil if lessons.empty?
+    lesson_plan = lessons.first.lesson_plan_html_url
+    expanded_card_resources = {"Lesson Plan" => lesson_plan}
+
+    lessons.each do |lesson|
+      break if expanded_card_resources.size >= 5
+      resources = lesson.resources
+
+      resources.each do |resource|
```

Comment:
Nit: I don't believe you use `resources` after this so you could simplify to `lesson.resources.each do |resource|`